### DISCUSSION TITLE
blog/cs/2024-04-13-Oracle-Join/

### DISCUSSION THREAD

**Discussion by giscus** [mingdodev/mingdodev.github.io#14] [Announcements]
# blog/cs/2024-04-13-Oracle-Join/

moonshot

https://mingdodev.github.io/blog/cs/2024-04-13-Oracle-Join/

**Comment by mikio999** [comment]
요즘 정처기에서 간단히만 훑고 간 Join인데 세세한 종류들이 있었군요! 글 잘 보고 갑니다 :)
